src/test/scala/advancepassengerinfo/health/HealthRouteTest.scala

```diff
@@ -0,0 +1,34 @@
+package advancepassengerinfo.health
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class HealthRouteTest extends AnyFlatSpec with Matchers with ScalatestRouteTest {
+
+  "HealthRoute" should "return OK status if process state is latest" in {
+    val mockProcessState = new ProcessState {
+      override def isLatest: Boolean = true
+    }
+
+    val route = HealthRoute(mockProcessState)
+
+    Get("/health-check") ~> route ~> check {
+      status shouldBe StatusCodes.OK
+    }
+  }
+
+  it should "return InternalServerError status if process state is not latest" in {
+    val mockProcessState = new ProcessState {
+      override def isLatest: Boolean = false
+    }
+
+    val route = HealthRoute(mockProcessState)
+
+    Get("/health-check") ~> route ~> check {
+      status shouldBe StatusCodes.InternalServerError
+      responseAs[String] shouldBe "KO"
```

Comment:
Is there a reason for returning the string "KO"?

Reply:
It is easy have a quick look on http what is health .

Reply:
What I meant was, what does KO mean? I don't see much reference to it on the internet other than europeans using it in slang to mean not ok. But I don't think it's going to be very obvious to english speakers what it means